runtime/dancebox/src/lib.rs

```diff
@@ -1429,6 +1506,9 @@ construct_runtime!(
         // InflationRewards must be after Session and AuthorInherent
         InflationRewards: pallet_inflation_rewards = 35,
 
+        // Treasury stuff.
```

Comment:
@augustocollado can you add pallet-treasury to the list of benchmarked pallets?

`frame_benchmarking::define_benchmarks!`

Reply:
both in dancebox and flashbox